admin/app/Models/User.php

```diff
@@ -2,28 +2,28 @@
 
 namespace App\Models;
 
-use Illuminate\Auth\Authenticatable;
-use Illuminate\Contracts\Auth\Access\Authorizable as AuthorizableContract;
-use Illuminate\Contracts\Auth\Authenticatable as AuthenticatableContract;
+use Illuminate\Contracts\Auth\MustVerifyEmail;
```

Comment:
Symbol 'App\Models\MustVerifyEmail' is declared but not used.

Reply:
I'm tempted to just delete User entirely, but worried that might break something in Laravel. I'll just apply your recommendations for now.